scene/main/viewport.cpp

```diff
@@ -896,13 +896,13 @@ void Viewport::_process_picking() {
 			if (camera_3d) {
 				Vector3 from = camera_3d->project_ray_origin(pos);
 				Vector3 dir = camera_3d->project_ray_normal(pos);
-				real_t far = camera_3d->get_far();
+				real_t camera_far = camera_3d->get_far();
```

Comment:
For symmetry with the existing code in `GLTFCamera`:

```suggestion
				real_t depth_far = camera_3d->get_far();
```